test/functionapp.test.ts

```diff
@@ -0,0 +1,99 @@
+import { ext } from '../src/extensionVariables';
+import { FunctionAppProvider } from '../src/tree/FunctionAppProvider';
+import * as fsUtil from '../src/utils/fs';
+import { longRunningTestsEnabled } from './global.test';
+
+// tslint:disable-next-line:max-func-body-length
+suite('Function App actions', async function (this: ISuiteCallbackContext): Promise<void> {
+    this.timeout(1200 * 1000);
+    const resourceGroupsToDelete: string[] = [];
+    const resourceName: string = fsUtil.getRandomHexString().toLowerCase();
+    const testAccount: TestAzureAccount = new TestAzureAccount();
+    suiteSetup(async function (this: IHookCallbackContext): Promise<void> {
+        if (!longRunningTestsEnabled) {
+            this.skip();
+        }
+        this.timeout(120 * 1000);
+        await testAccount.signIn();
+        ext.tree = new AzureTreeDataProvider(FunctionAppProvider, 'azureFunctions.startTesting', undefined, testAccount);
+    });
+    suiteTeardown(async function (this: IHookCallbackContext): Promise<void> {
+        if (!longRunningTestsEnabled) {
+            this.skip();
+        }
+        this.timeout(1200 * 1000);
+        const client: ResourceManagementClient = getResourceManagementClient(testAccount);
+        for (const resourceGroup of resourceGroupsToDelete) {
+            if (await client.resourceGroups.checkExistence(resourceGroup)) {
+                console.log(`Deleting resource group "${resourceGroup}"...`);
+                await client.resourceGroups.deleteMethod(resourceGroup);
+                console.log(`Resource group "${resourceGroup}" deleted.`);
+            } else {
+                // If the test failed, the resource group might not actually exist
+                console.log(`Ignoring resource group "${resourceGroup}" because it does not exist.`);
+            }
+        }
+    });
+    test('createFunctionApp', async () => {
+        resourceGroupsToDelete.push(resourceName);
+        const testInputs: string[] = [resourceName, '$(plus) Create new resource group', resourceName, '$(plus) Create new storage account', resourceName, 'East US'];
+        ext.ui = new TestUserInput(testInputs);
+        await vscode.commands.executeCommand('azureFunctions.createFunctionApp', testAccount.getSubscriptionId());
+        const client: WebSiteManagementClient = getWebsiteManagementClient(testAccount);
+        const createdApp: WebSiteManagementModels.Site = await client.webApps.get(resourceName, resourceName);
+        assert.ok(createdApp);
+    });
+    test('stopFunctionApp', async () => {
+        ext.ui = new TestUserInput([resourceName]);
+        await vscode.commands.executeCommand('azureFunctions.stopFunctionApp');
+        const client: WebSiteManagementClient = getWebsiteManagementClient(testAccount);
+        const createdApp1: WebSiteManagementModels.Site = await client.webApps.get(resourceName, resourceName);
```

Comment:
Just call it createdApp (this applies to all of the tests.)  They aren't being numbered anyway and they're all within their own scope, so it's okay to reuse the variable name :)